chef/cookbooks/cinder/templates/default/cinder.conf.erb

```diff
@@ -1644,7 +1644,7 @@ nfs_shares_config=/etc/cinder/nfs_shares-<%= backend_id %>
 
 # Mount options passed to the nfs client. See section of the nfs man page for
 # details. (string value)
-#nfs_mount_options = <None>
+nfs_mount_options = <%= volume['netapp']['nfs_mount_options'] %>
```

Comment:
> @vuntz shall we need to make this optional somehow?

What do you mean? Is this related to the fact that the attribute is not required? I think that would be fine because then it means `<%= nil %>` which should result in an empty string.

Btw, some quick comments:

- I'm surprised hound doesn't complain, but please use `"` insetad of `'`
- don't remove the commented out default value, we always leave this for reference